cmd/manager/main.go

```diff
@@ -4,13 +4,13 @@ import (
 	"context"
 	"flag"
 	"fmt"
-	"os"
-	"runtime"
-
 	"github.com/mattermost/mattermost-operator/pkg/apis"
 	"github.com/mattermost/mattermost-operator/pkg/controller"
 	"github.com/mattermost/mattermost-operator/pkg/log"
 	"github.com/mattermost/mattermost-operator/version"
+	"os"
```

Comment:
nit: these three imports make a bit more sense near the top where they originally were.

Reply:
Yes, agreed, my IDE has its own opinion - tries to order alphabetically all imports